### PR TITLE
1542 template in acceptance tests

### PR DESCRIPTION
**Pull request by travissarles**

### CODE REVIEW

**Review by joke1196 — APPROVED** [review]

File: acceptance-tests/src/test/scala/ch/renku/acceptancetests/model/projects.scala [hunk 1]
```diff
@@ -66,17 +69,19 @@ object projects {
       val desc =
         maybeDescription.getOrElse(prefixParagraph("An automatically generated project for testing: ").generateOne)
       val readmeTitle = s"test ${now.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH-mm-ss"))}"
-      val template    = maybeTemplate.getOrElse("Renku/python-minimal")
-      ProjectDetails(title, visibility, desc, readmeTitle)
+      val template    = Template(maybeTemplate.getOrElse(Refined.unsafeApply("Renku/python-minimal")))
```
Comment (jachro) [inline acceptance-tests/src/test/scala/ch/renku/acceptancetests/model/projects.scala:72]: I guess you don't need to wrap the `"Renku/python-minimal"` into `Refined.unsafeApply`. Simply add 
`import eu.timepit.refined.auto._`

File: acceptance-tests/src/test/scala/ch/renku/acceptancetests/model/projects.scala [hunk 2]
```diff
@@ -66,17 +69,19 @@ object projects {
...
         val readmeTitle = "flights tutorial"
-        ProjectDetails(Refined.unsafeApply(readmeTitle),
-                       Visibility.Public,
-                       Refined.unsafeApply("A renku tutorial project."),
-                       readmeTitle
+        ProjectDetails(
+          Refined.unsafeApply(readmeTitle),
+          Visibility.Public,
+          Refined.unsafeApply("A renku tutorial project."),
+          Template(Refined.unsafeApply("This template isn't used")),
```
Comment (jachro) [inline acceptance-tests/src/test/scala/ch/renku/acceptancetests/model/projects.scala:83]: See the previous comment.

File: acceptance-tests/src/test/scala/ch/renku/acceptancetests/model/projects.scala (hunk 1 above)
Comment (travissarles) [reply acceptance-tests/src/test/scala/ch/renku/acceptancetests/model/projects.scala:72]: I thought it had to be simpler! :) There was some other refactoring we wanted to do so I'll try to remember to fix that as well.

File: acceptance-tests/src/test/scala/ch/renku/acceptancetests/tooling/ExistingProjectSpecData.scala [hunk 3]
```diff
@@ -12,8 +12,16 @@ trait ExistingProjectSpecData {
...
   protected lazy val existingProjectDetails: Option[ProjectDetails] = {
     Option(getProperty("extant")) orElse sys.env.get("RENKU_TEST_EXTANT_PROJECT") match {
-      case Some(s) => Some(ProjectDetails(Refined.unsafeApply(s), Visibility.Public, Refined.unsafeApply("unused"), s))
-      case None    => None
+      case Some(readMeTitle) =>
+        Some(
+          ProjectDetails(Refined.unsafeApply(readMeTitle),
+                         Visibility.Public,
+                         Refined.unsafeApply("unused"),
+                         Template(Refined.unsafeApply("Not used")),
```
Comment (jachro) [inline acceptance-tests/src/test/scala/ch/renku/acceptancetests/tooling/ExistingProjectSpecData.scala:20]: See comments above

**Pull request merged**